src/ripple/app/ledger/LedgerReplayTask.h

```diff
@@ -0,0 +1,196 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2020 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#ifndef RIPPLE_APP_LEDGER_LEDGERREPLAYTASK_H_INCLUDED
+#define RIPPLE_APP_LEDGER_LEDGERREPLAYTASK_H_INCLUDED
+
+#include <ripple/app/ledger/InboundLedger.h>
+#include <ripple/app/ledger/impl/TimeoutCounter.h>
+#include <ripple/app/main/Application.h>
+
+#include <memory>
+#include <vector>
+
+namespace ripple {
+class InboundLedgers;
+class Ledger;
+class LedgerDeltaAcquire;
+class LedgerReplayer;
+class SkipListAcquire;
+namespace test {
+class LedgerReplayClient;
+}  // namespace test
+
+class LedgerReplayTask final
+    : public TimeoutCounter,
+      public std::enable_shared_from_this<LedgerReplayTask>,
+      public CountedObject<LedgerReplayTask>
+{
+public:
+    class TaskParameter
+    {
+    public:
+        // set on construct
+        InboundLedger::Reason reason;
+        uint256 finishHash;
+        std::uint32_t totalLedgers;  // including the start and the finish
+
+        // to be updated
+        std::uint32_t finishSeq = 0;
+        std::vector<uint256> skipList = {};  // including the finishHash
+        uint256 startHash = {};
+        std::uint32_t startSeq = 0;
+        bool full = false;
```

Comment:
All of these member names should end with `_`, even if they are public.

Reply:
thanks, changed.